Documentation/markdown/GitProcess.md

```diff
@@ -0,0 +1,15 @@
+# Git Procedures 
+
+Here you will find description about Git procedures you need working with OpenInfraPlatform
```

Comment:
Add a link to / include the top figure from [here](https://www.c-sharpcorner.com/article/git-and-github-version-control-local-and-remote-repository/).

Reply:
Included Git_Workflow.png